gui/freehand.py

```diff
@@ -359,6 +363,8 @@ def motion_notify_cb(self, tdw, event, fakepressure=None):
         pressure = event.get_axis(Gdk.AxisUse.PRESSURE)
         xtilt = event.get_axis(Gdk.AxisUse.XTILT)
         ytilt = event.get_axis(Gdk.AxisUse.YTILT)
+        viewzoom = tdw.scale
+        viewrotation = tdw.rotation
```

Comment:
Yes, you're right with the other programs point. It's subtle and evil to change the meanings of tilt on them. Let's keep the meaning as at present.

The right place to hang this, if any optimization is needed, is `Brush::stroke_to()` in `brush.hpp`.